src/crypto-api/index.ts

```diff
@@ -418,6 +418,12 @@ export interface CryptoApi {
      */
     getEncryptionInfoForEvent(event: MatrixEvent): Promise<EventEncryptionInfo | null>;
 
+    /**
+     * All users *whose device lists we are tracking* are flagged as needing a
+     * key query. Users whose devices we are not tracking are ignored.
+     */
+    markAllTrackedUsersAsDirty(): Promise<void>;
```

Comment:
Are you sure this needs exposing to the application? If so, could you share some guidance on when an application should call it? Also, could you have a look through the whole of `CryptoApi` and see if there isn't a better place it fits in the order of methods?

Reply:
Seems like it only needed to be in `SyncCryptoCallbacks` which it already was - removed.